COLLABORATOR_GUIDE.md

```diff
@@ -345,7 +345,7 @@ Additionally:
 
```

Comment:
Up there (out of context) 309 & 311 are almost dups. also I'd pull "Please never use GitHub's green" down here.